code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm

```diff
@@ -260,28 +271,35 @@
 				if(!istype(X,/obj/item/forbidden_book))
 					continue
 				var/obj/item/forbidden_book/FB = X
-				FB.charge++
+				FB.charge += 2
```

Comment:
Remind you, this change will conflict with #3451

Reply:
Yes, #3451 should be implemented after this one.